packages/schematics/angular/application/other-files/app.component.spec.ts.template

```diff
@@ -3,16 +3,16 @@ import { RouterTestingModule } from '@angular/router/testing';<% } %>
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
-  beforeEach(async(() => {
+  beforeEach(async(() =>
```

Comment:
Can you convert these to proper async anonymous functions instead of the custom async function call?  Please keep the blocks though to simplify user additions to these functions.

Reply:
You mean using the native async keyword instead of the custom angular async function right?

Reply:
yes

Reply:
Done